code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -192,8 +192,10 @@
 		var/mob/living/carbon/human/H = M
 		if(H.species.name == src.species.name)
 			return
-		to_chat(H, "\red Your nose begins to bleed...")
-		H.drip(1)
+
+		H << "\red Your nose begins to bleed..."
```

Comment:
We need to phase out the \reds wherever possible. Also this needs to be a to_chat().